ApollosProject/apollos-apps
# feat: UI 2.0 Image Media

## Motivation
We wanted to port over the existing `ConnectedImage` component and clean it up a bit if there were quick opportunities. 

## Solution
Rename the component to `ImageMedia` in order to better match the API schema.

Rename the `maintainAspectRatio` property to `autoSize` to drive clarity.

`autoSize` is `true` by default in order to act _closer_ to the `img` tag in Html

Deprecate the `ImageComponent` prop in favor or a specified styled Animated.Image tag.

## Testing
Test coverage is at 100%

Stories have been written to demonstrate the basic functionality and the aspect ratio logic

**To Test ::**
1. set `PROJECT_VARIANT=storybook` in your `env`
2. `yarn && yarn start`
3. Open a new terminal `yarn ipados`
4. View the storybook documentation
5. Run `yarn test ui-kit` from the `templates/mobile` directory in order to see test coverage
